core/src/main/kotlin/com/justai/jaicf/channel/invocationapi/InvocationRequest.kt

```diff
@@ -0,0 +1,44 @@
+package com.justai.jaicf.channel.invocationapi
+
+import com.justai.jaicf.api.BotRequest
+import com.justai.jaicf.api.EventBotRequest
+import com.justai.jaicf.api.QueryBotRequest
+
+val BotRequest.invocationApi get() = this as? InvocationRequest
```

Comment:
`invocationApi` is more suitable for some sort of API, not an data class that contains a request's details.

Reply:
Renamed to `invocationRequest`